src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Features/ConfirmRolesAndResponsibilities.cs

```diff
@@ -0,0 +1,106 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using SFA.DAS.ApprenticeCommitments.Api.Controllers;
+using SFA.DAS.ApprenticeCommitments.Data.Models;
+using System.Net;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+
+namespace SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests.Features
+{
+    [Binding]
+    [Scope(Feature = "ConfirmRolesAndResponsibilities")]
+    public sealed class ConfirmRolesAndResponsibilities
+    {
+        private readonly Fixture _fixture = new Fixture();
+        private readonly TestContext _context;
+        private readonly Apprentice _apprentice;
+        private readonly Apprenticeship _apprenticeship;
+        private bool? RolesAndResponsibilitiesCorrect { get; set; }        
+        private string endpoint;
+        private object command;
```

Comment:
`endpoint` and `command` were there because I was using one steps file for multiple features.  If we're going to have this feature bound to its own step file then they aren't necessary - the `endpoint` can simply be inlined, and the `command` can be turned into a bool, with the `ConfirmRolesAndResponsibilitiesRequest` created inside `WhenWeSendTheConfirmation`